jar-service/pom.xml

```diff
@@ -200,8 +200,8 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<target>1.6</target>
-					<source>1.6</source>
+					<target>1.7</target>
+					<source>1.7</source>
```

Comment:
I initially used multiple-catch statements, but then when I was testing and running the server in Eclipse (yes ... still in Eclipse :)), this part of the code was failing in the JVM, so I reverted back to separate catches.  Here is the corresponding issue to change it everywhere #365